akka-cluster/src/main/resources/reference.conf

```diff
@@ -487,6 +487,12 @@ akka.cluster.split-brain-resolver.keep-oldest {
 akka.cluster.split-brain-resolver.lease-majority {
   lease-implementation = ""
 
+  # The identifier of the lease.
+  #  lease-name = "<service-name>-akka-sbr"
+  # When not set, it default to
+  #  lease-name = "<actor-system-name>-akka-sbr"
+  #lease-name = ""
```

Comment:
In Akka we typically use `""` as the undefined value. Better to have all properties in reference.conf and avoid `hasPath` stuff (some tooling depend on that such as config checker, and maybe also IntelliJ)